internal/manifests/api/interface.go

```diff
@@ -8,13 +8,13 @@ import (
 	operations "github.com/openshift/assisted-service/restapi/operations/manifests"
 )
 
-//go:generate mockgen -package api -destination mock_manifests_api.go . ManifestsAPI
+//go:generate mockgen --build_flags=--mod=mod --build_flags=--mod=mod -package api -destination mock_manifests_api.go . ManifestsAPI
```

Comment:
and here why the same argument needs to appear twice?

Reply:
typo